data/singhmohit14072002.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Mohit Singh",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "OpenSource  | DevOps | Docker | Blogger **Hashnode**",
+  "avatar": "https://github.com/singhmohit14072002.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/https://twitter.com/MohitDevops",
```

```suggestion
      "url": "https://twitter.com/MohitDevops",
```
Please update the url